hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -498,10 +498,10 @@ public static void validateVolumeName(String volumeName) throws OMException {
   /**
    * Verify bucket name is a valid DNS name.
    */
-  public static void validateBucketName(String bucketName)
+  public static void validateBucketName(String bucketName, boolean isNamespaceS3RuleCompliant)
```

Comment:
I think the name `isNamespaceS3RuleCompliant` could be improved.  (`strictS3` may be one choice, but feel free to find another one.)

Reply:
yeah definitely! I changed it to strictS3.  it's very good!